Assets/Scripts/VR/VRInteraction.cs

```diff
@@ -240,22 +240,21 @@ void showIndicator(GameObject controllerObject, bool showIndicator)
 
                 if (interactionTrigger())
                 {
-                    if (mainUIController.IsDrawing && annotationTool != null && !EventSystem.current.IsPointerOverGameObject())
+                    if (menuController.IsDrawing && annotationTool != null && !EventSystem.current.IsPointerOverGameObject())
                     {
                         // Seems like a crazy way to determine if the UI is in the way, but after multiple attempts at a more
                         // elegant solution, settled on this since it works.
 
-                        if (FindObjectOfType<LaserPointer>().GetComponent<LineRenderer>().enabled)
-
-                        {
+                        //if (FindObjectOfType<LaserPointer>().GetComponent<LineRenderer>().enabled)
```

Comment:
can we delete these lines of commented code?

Reply:
See previous comment on lasers :)